### DISCUSSION TITLE
recreate the database ps3

### DISCUSSION THREAD

**Discussion by caslu77** [bucanero/apollo-ps3#73] [Q&A] [answered]
Every time I open a trophy in Apollo, when I leave the Ps3 keeps asking me to recreate the database, there's something wrong, right

**Comment by bucanero** [comment, marked as answer]
if you modify or edit trophies, you need to rebuild the database or the changes won't show up in the XMB.

So Apollo is asking to schedule a rebuild so the trophies are updated in the internal database. You can skip it, but the results will be unknown.

**Reply by caslu77** [reply]
is there a risk of ruining the ps3 by recreating the database all the time?

**Reply by caslu77** [reply]
I can not recreate the databases and log into psn to sync the trophies, will they show up?

**Reply by bucanero** [reply]
as I said, if you don't rebuild the database, the results are unknown... you can try to sync and see if it works or not. 

in any case, if you're worried about the database integrity, I suggest to avoid any trophy edit or sync. (These changes are made directly to the data files, not like an official unlock in-game)
